src/main/java/io/cloudsoft/terraform/TerraformSshDriver.java

```diff
@@ -73,15 +74,16 @@ protected String getLogFileLocation() {
         return getStateFilePath();
     }
 
+    final String ARM_ARCH_PATTERNS = "(arm|aarch)\\w*";
     public String getOsTag() {
         OsDetails os = getLocation().getOsDetails();
         // If no details, assume 64-bit Linux
         if (os == null) return "linux_amd64";
         // If not Mac, assume Linux
         String osType = os.isMac() ? "darwin" : "linux";
-        String archType = os.is64bit() ?
-                os.getArch().toLowerCase().contains("arm") ? "arm64" : "amd64":
-                os.getArch().toLowerCase().contains("arm") ? "arm" : "386";
+        String archType = os.is64bit() ? // temporary, remove after Juan's changes in brooklyn-core
+                Pattern.matches(ARM_ARCH_PATTERNS, os.getArch().toLowerCase()) ? "arm64" : "amd64":
```

Comment:
this could be now replaced with `os.isArm`.
don't worry for replace it

Reply:
@jcabrerizo  make the changes on `main` and send the PR, I will merge it in my branch.

Reply:
PR: https://github.com/cloudsoft/brooklyn-terraform/pull/55

Reply:
@jcabrerizo thx